blockchain/blockchain.go

```diff
@@ -50,7 +50,7 @@ type Reader interface {
 
 var (
 	ErrParentDoesNotMatchHead = errors.New("block's parent hash does not match head block hash")
-	supportedStarknetVersion  = semver.MustParse("0.13.0")
+	SupportedStarknetVersion  = semver.MustParse("0.13.0")
```

Comment:
I feel like we should add a `SupportedVersion() semver.Version` to the `blockchain.Reader` interface, but can't come up with a good reason. I guess we can deal with the exported global variable when it actually starts getting in the way. Was that your thinking as well?

Reply:
I think a package level exported constant is fine. Since different instances won't have support for different versions, I don't think need to add `SupportedVersion() semver.Version`.